cds/modules/deposit/api.py

```diff
@@ -602,6 +602,9 @@ def _publish_videos(self):
         # update project video references
         self._update_videos(refs_old, refs_new)
 
+        # ensure the published videos are saved in the DB
+        db.session.commit()
```

Comment:
I would try to execute this line after the publishing of the videos and project, before [this line](https://github.com/CERNDocumentServer/cds-videos/blob/main/cds/modules/deposit/api.py#L617) it could be a good candidate, WDYT?